packages/block-library/src/navigation-link/edit.js

```diff
@@ -797,8 +817,13 @@ export default function NavigationLinkEdit( {
 										<>
 											<span>
 												{
-													/* Trim to avoid trailing white space when the placeholder text is not present */
-													`${ label } ${ placeholderText }`.trim()
+													// Some attributes are stored in an escaped form due to the escaping model
```

Comment:
```suggestion
													// Some attributes are stored in an escaped form.  – It's a legacy issue, ideally they would be stored in a raw, unescaped form.
```

Reply:
Thanks for suggestion. I've updated in https://github.com/WordPress/gutenberg/pull/41063/commits/0d0de3a1e3916002f9f98102ef6d4bd90334a070.